components/script/lib.rs

```diff
@@ -2,15 +2,15 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 
-#![feature(register_tool)]
+#![cfg_attr(crown, feature(register_tool))]
 #![deny(unsafe_code)]
 #![doc = "The script crate contains all matters DOM."]
 // Register the linter `crown`, which is the Servo-specific linter for the script
 // crate. Issue a warning if `crown` is not being used to compile, but not when
 // building rustdoc or running clippy.
-#![register_tool(crown)]
-#![cfg_attr(any(doc, clippy, feature = "cargo-clippy"), allow(unknown_lints))]
-#![deny(crown_is_not_used)]
```

Comment:
See here we abuse unknown_lints lint to get warning if crown is not used.

Reply:
Let's also keep warning if crown is not used.

Reply:
done